.github/workflows/test.yml

```diff
@@ -42,9 +43,8 @@ jobs:
         if: (! contains(matrix.python-version, 'pypy'))
         run: |
           coverage xml
-          coverage report --show-missing --fail-under=100
```

Comment:
Removing this line here as it requires every single CI build to have 100% coverage. This isn't quite right in cases when there are Python version-specific lines of code (which we have in `toolz`). 

Instead, let's merge coverage reports across all CI build with codecov to make sure that, summed across all Python versions, we have 100% coverage.